android/app/src/main/AndroidManifest.xml

```diff
@@ -49,13 +49,7 @@
             <action android:name="android.intent.action.VIEW"/>
             <category android:name="android.intent.category.DEFAULT"/>
             <category android:name="android.intent.category.BROWSABLE"/>
-            <data android:scheme="https" android:host="expensify.cash" />
-        </intent-filter>
-        <intent-filter>
-            <action android:name="android.intent.action.VIEW"/>
-            <category android:name="android.intent.category.DEFAULT"/>
-            <category android:name="android.intent.category.BROWSABLE"/>
-            <data android:scheme="https" android:host="www.expensify.cash" />
```

Comment:
We previously had a separate filter for `expensify.cash` and `www.expensify.cash`, shouldn't we add one here for `www.new.expensify.com`? Context for this in https://github.com/Expensify/Expensify.cash/pull/2052#issuecomment-806265667

Reply:
No idea - @marcaaron?